repos/system_upgrade/common/files/upgrade_paths.json

```diff
@@ -21,6 +21,11 @@
         "default": {
             "8": ["9"],
             "9": ["10"]
+        },
+        "_virtual_versions": {
+            "8": "8.10",
+            "9": "9.5",
+            "10": "10.0"
```

Comment:
Can be changed later, but based on the merge time, we can virtual versions already 9.6, 10.0. It would be possible even 9.7, 10.1 as Centos Stream contains already changes for new minor releases, we can stay with 9.6, 10.0.

This does not block the merge, it's a nitpick now.

Reply:
I have modified virtual version for CentOS 9 to 9.6. Virtual version for CentOS 10 is unchanged.